package.json

```
@@ -37,7 +37,7 @@
     "broccoli-merge-trees": "^2.0.0 || ^3.0.0",
     "ember-cli-babel": "^6.8.2",
     "ember-cli-is-package-missing": "^1.0.0",
-    "ember-cookies": "^0.3.0 || ^0.4.0",
+    "ember-cookies": "^0.3.0 || ^0.5.0",
```

```suggestion
    "ember-cookies": "^0.5.0",
```

The `sameSite` option is only available with >= 0.5.0